test/distributed/test_c10d_nccl.py

```diff
@@ -4137,6 +4137,7 @@ def _wait_process(self, rank, timeout):
             return None
 
 
+@skip_but_pass_in_sandcastle
```

Comment:
iiuc this macro would only skip in sandcastle, not oss CI?  or am i wrong

Reply:
This will skip on GitHub and pass on sandcastle.